README.md

```diff
@@ -1,2 +1,10 @@
 # XebiaLabs-Appium-ReactNative
-Guidance on integrating Kobiton service into the mobile app build pipeline: XebiaLabs, Appium and ReactNative.
+
+XebiaLabs is a service that develops Continuous Delivery and DevOps software to provide companies greater control over their software delivery process. In XL Release, you can plan, automate, and optimize your pipeline.
+
+The guidelines in this repo will show you how to plan a release with the XebiaLabs service XL release as well as how to trigger a Jenkins build from a task on XebiaLabs.
```

Comment:
How about `This guideline will show you to include executing tests on Kobiton devices in your release plan in XebiaLabs  XL Release`